### ISSUE TITLE
mobile webgl image decals

### ISSUE THREAD

**Issue by danielemichieletto** [processing/p5.js#4183]
On iOs mobile devices, 2 images applied on very close surfaces are displayed intermittently.
On Android and desktop is OK

#### Nature of issue?

- [ ] Found a bug
- [X ] Existing feature enhancement
- [ ] New feature request

#### Most appropriate sub-area of p5.js?

- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ X] WebGL
- [ ] Other (specify if possible)

#### Which platform were you using when you encountered this?

- [X ] Mobile/Tablet (touch devices)
- [ ] Desktop/Laptop
- [ ] Others (specify if possible)

#### Details about the bug: 

- p5.js version: 0.9
- Web browser and version: Safari app and Chrome app
- Operating System: iOs 13.2.3
- Steps to reproduce this:
https://editor.p5js.org/daniele.michieletto/sketches/jSySszzQ4

**Comment by aferriss** [comment]
This looks like maybe a z-fighting issue to me. I took a recording on an iPhone XR with safari 13.2.3

https://www.dropbox.com/s/501tl7jo6e2rnyq/RPReplay_Final1575658616.MP4?dl=0

**Comment by stalgiag** [comment]
Yes this seems like a depth precision issue. Not necessarily something we can fix on our end, but we can make it easier to identify these problems and work with depth precision, near, and far values.

I think that the current way of selecting default near and far clipping values might be a bit too opaque. [It is automatically calculated](https://github.com/processing/p5.js/blob/master/src/webgl/p5.Camera.js#L1068) based on size of canvas and field of view. This is good in some scenarios but I have seen this create confusion, especially when someone is making a windowWidth/windowHeight sketch. In different sized windows, this can cause sketches to have unexpected overlap or occlude near or far objects. 

Perhaps we should give near and far non-calculated default values, and figure out a way to warn people when they may be drawing outside of it? This second part seems a bit involved, maybe part of the FES.
